### ISSUE TITLE
The online version seems to be down

### ISSUE THREAD

**Issue by zoechi** [thosakwe/pub_mediator#3]

**Comment by thosakwe** [comment]
Yes, I've taken it down. I wasn't approved for AdSense until a few months after putting the site up, so I was pretty much just losing money.

When I have free time this weekend, I'll see if I can get the site up again.
